### PR TITLE
Fixed client tests

### PR DESCRIPTION
**Pull request by DanWaldman**
Now client tests compare between response and entire HTML file #107

### CODE REVIEW

**Comment by shakedhagag** [comment]
Nice work

**Review by shakedhagag — APPROVED** [review]
LGTM

**Review by razashtar93 — APPROVED** [review]
Good job!

File: AcadeMeData/tests/test_client.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
     request = client.get('')
     response = views.homePage(request)
+    expected_html = render_to_string('landing/homepage.html', request=request)
     assert response.status_code == 200
+    assert response.content.decode() == expected_html
 
 
 def test_contactus(client):
     request = client.get('contact_us/')
     response = views.contact(request)
```
Comment (Yarboa) [inline AcadeMeData/tests/test_client.py:15]: There are two checks here
response=client.get('contact_us/')
Line 15 is another test but it is not relevant here,
My mistake in the review it was approved

File: AcadeMeData/tests/test_client.py [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+    expected_html = render_to_string('landing/contact_us.html', request=request)
     assert response.status_code == 200
```
Comment (Yarboa) [inline AcadeMeData/tests/test_client.py:17]: It should be split to two tests once for client request, the other for the view generate the HTML.

**Review by Yarboa — APPROVED** [review]

File: AcadeMeData/tests/test_client.py (hunk 2 above)
Comment (DanWaldman) [reply AcadeMeData/tests/test_client.py:21]: Done

**Review by DeanCivlin — APPROVED** [review]

**Review by DanielItkes — APPROVED** [review]

File: AcadeMeData/tests/test_client.py (hunk 1 above)
Comment (DanWaldman) [reply AcadeMeData/tests/test_client.py]: Split to two test files, and commited the changes you mentioned above. @Yarboa

File: AcadeMeData/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from django.template.loader import render_to_string
+import pytest
+
+
+@pytest.mark.django_db
+def test_homepage_html(client):
+    response = client.get('')
```
[the remark this answers is not in the document]
Comment (DanWaldman) [reply AcadeMeData/tests/test_views.py]: like this? @Yarboa

**Comment by DanWaldman** [comment]
@Omeramsc could this get merged? @Yarboa  approved

**Comment by DanWaldman** [comment]
> @Omeramsc could this get merged? @Yarboa  approved

????@Yarboa @Omeramsc

**Review by Omeramsc — APPROVED** [review]
LGTM
@DanWaldman Do note we are not always available. :)

**Pull request merged**